broker/src/main/java/org/apache/rocketmq/broker/processor/AdminBrokerProcessor.java

```diff
@@ -519,6 +523,12 @@ private synchronized RemotingCommand updateAndCreateTopic(ChannelHandlerContext
         }
         long executionTime = System.currentTimeMillis() - startTime;
         LOGGER.info("executionTime of create topic:{} is {} ms" , topic, executionTime);
+        Attributes attributes = BrokerMetricsManager.newAttributesBuilder()
+                .put(LABEL_REQUEST_IS_SUCCESS, response.getCode() == ResponseCode.SUCCESS)
```

Comment:
If the code reaches this point, it appears that the request must have been successful. Because the failed request has already been returned in advance

Reply:
Yes, I overlooked the issue with the logical branches. The latest changes incorporate the metric recording logic into the finally block, ensuring it executes under all circumstances.